app/controllers/programs_controller.rb

```diff
@@ -20,7 +20,7 @@ def create
   end
 
   def new
-    if current_user.admin?
+    if current_user.try(:admin?)
```

Comment:
Please read through the lesson on [Devise authorization](https://learn.launchacademy.com/lessons/devise-authorization) for examples of how to limit access to particular controller actions in a DRYer way.